ocaml/xapi/xapi.ml

```diff
@@ -376,6 +373,7 @@ let bring_up_management_if ~__context () =
     ) ;
     (* Start the Host Internal Management Network, if needed. *)
     Xapi_network.check_himn ~__context ;
+    update_certificates ~__context () ;
```

Comment:
Maybe put this after line 362. That's when we are in the case that stunnel will definitely have started, and we have more reasons to log with `error` if the certs can not be synced.

Reply:
Don't we want to regenerate the bundle _before_ starting the stunnel server? I assume this is one of the things `update_certificates` does

Reply:
I assumed that it just updates DB records.

Reply:
The function name could be made clearer.

Reply:
The function updates the database record to match the contents of /etc/xensource/xapi_ssl.pem